testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -230,6 +249,13 @@ class MiniAppDisplayActivity : BaseActivity() {
         super.onActivityResult(requestCode, resultCode, data)
         if (requestCode == externalWebViewReqCode && resultCode == Activity.RESULT_OK) {
             data?.let { intent -> sampleWebViewExternalResultHandler.emitResult(intent) }
+        } else if (requestCode == filePickingReqCode && resultCode == Activity.RESULT_OK) {
+            data?.let { intent ->
+                if (filePathHostCallback == null) return
+
+                val result: Uri? = intent.data
+                filePathHostCallback?.onReceiveValue(arrayOf(result!!))
```

Comment:
I think you're using `filePathHostCallback?` so you can remove `if (filePathHostCallback == null) return`
-> I'm not sure so let's test it 🙇‍♂️

Reply:
I think, you are right..